From 4c3ad08552bcb98ef76da570043a7d4052f8ada3 Mon Sep 17 00:00:00 2001
From: Remil Ram <remilr@codeaurora.org>
Date: Fri, 24 Jul 2020 11:25:45 -0700
Subject: [PATCH] remove-journal-from-systemctl

---
 src/systemctl/systemctl.c | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/src/systemctl/systemctl.c b/src/systemctl/systemctl.c
index 3e4fc46..aef5600 100644
--- a/src/systemctl/systemctl.c
+++ b/src/systemctl/systemctl.c
@@ -4612,20 +4612,6 @@ static void print_status_info(
                                           i->id, bus_error_message(&error, r));
         }
 
-        if (i->id && arg_transport == BUS_TRANSPORT_LOCAL)
-                show_journal_by_unit(
-                                stdout,
-                                i->id,
-                                arg_output,
-                                0,
-                                i->inactive_exit_timestamp_monotonic,
-                                arg_lines,
-                                getuid(),
-                                get_output_flags() | OUTPUT_BEGIN_NEWLINE,
-                                SD_JOURNAL_LOCAL_ONLY,
-                                arg_scope == UNIT_FILE_SYSTEM,
-                                ellipsized);
-
         if (i->need_daemon_reload)
                 warn_unit_file_changed(i->id);
 }
@@ -5442,9 +5428,6 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
                                 if (!eq)
                                         continue;
 
-                                if (!journal_field_valid(p, eq - (const char*) p, false))
-                                        continue;
-
                                 str = malloc(sz + 1);
                                 if (!str)
                                         return log_oom();
-- 
2.7.4

