From 91d95a0361afda0e0f3707be5142f6aa8039c500 Mon Sep 17 00:00:00 2001
From: Remil Ram <remilr@codeaurora.org>
Date: Fri, 24 Jul 2020 11:43:56 -0700
Subject: [PATCH] src-shared-meson-build

---
 src/shared/meson.build | 16 +---------------
 1 file changed, 1 insertion(+), 15 deletions(-)

diff --git a/src/shared/meson.build b/src/shared/meson.build
index 063d157..44445d9 100644
--- a/src/shared/meson.build
+++ b/src/shared/meson.build
@@ -54,8 +54,6 @@ shared_sources = files('''
         daemon-util.h
         dev-setup.c
         dev-setup.h
-        dissect-image.c
-        dissect-image.h
         dm-util.c
         dm-util.h
         dns-domain.c
@@ -99,10 +97,6 @@ shared_sources = files('''
         install.h
         ip-protocol-list.c
         ip-protocol-list.h
-        journal-importer.c
-        journal-importer.h
-        journal-util.c
-        journal-util.h
         json-internal.h
         json.c
         json.h
@@ -118,12 +112,8 @@ shared_sources = files('''
         lockfile-util.c
         lockfile-util.h
         log-link.h
-        logs-show.c
-        logs-show.h
         loop-util.c
         loop-util.h
-        machine-image.c
-        machine-image.h
         machine-pool.c
         machine-pool.h
         main-func.h
@@ -172,8 +162,6 @@ shared_sources = files('''
         tmpfile-util-label.h
         tomoyo-util.c
         tomoyo-util.h
-        udev-util.c
-        udev-util.h
         uid-range.c
         uid-range.h
         unit-file.c
@@ -293,15 +281,13 @@ libshared_static = static_library(
 
 libshared = shared_library(
         libshared_name,
-        libudev_sources,
         include_directories : includes,
         link_args : ['-shared',
                      '-Wl,--version-script=' + libshared_sym_path],
         link_whole : [libshared_static,
                       libbasic,
                       libbasic_gcrypt,
-                      libsystemd_static,
-                      libjournal_client],
+                      libsystemdq_static],
         c_args : ['-fvisibility=default'],
         dependencies : libshared_deps,
         install : true,
-- 
2.7.4

