From e66ad37a961fc53436240e21a7a78fc06b8a5c45 Mon Sep 17 00:00:00 2001
From: Sourabh Banerjee <sbanerje@codeaurora.org>
Date: Tue, 20 Aug 2019 11:00:42 +0530
Subject: [PATCH] app: aboot: mount rootfs as read-write

---
 app/aboot/aboot.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/aboot/aboot.c b/app/aboot/aboot.c
index 07b5687..5e04803 100755
--- a/app/aboot/aboot.c
+++ b/app/aboot/aboot.c
@@ -196,7 +196,7 @@ static const char *resume = " resume=/dev/mmcblk0p";
 #endif
 
 #ifdef INIT_BIN_LE
-static const char *sys_path_cmdline = " rootwait ro init="INIT_BIN_LE;
+static const char *sys_path_cmdline = " rootwait rw init="INIT_BIN_LE;
 #else
 static const char *sys_path_cmdline = " rootwait ro init=/init";
 #endif
-- 
1.9.1

